lib/rubycritic/commands/status_reporter.rb

```diff
@@ -0,0 +1,43 @@
+module Rubycritic
+  module Command
+    class StatusReporter
+      attr_reader :status, :status_message
+      SUCCESS = 0
+      SCORE_BELOW_MINIMUM = 1
```

Comment:
Lets use those constants in our features above instead of the magic numbers  0 and 1 directly.